teamscale-client/src/main/java/com/teamscale/client/ClusteredTestDetails.java

```diff
@@ -12,23 +12,17 @@ public class ClusteredTestDetails extends TestDetails {
 	 */
 	public String clusterId;
 
-	public ClusteredTestDetails(String uniformPath, String sourcePath, String content, String clusterId) {
-		super(uniformPath, sourcePath, content);
-		this.clusterId = clusterId;
-	}
-
 	/**
-	 * Creates clustered test details with the given additional {@link TestData}.
-	 * <p>
-	 * Use this to easily mark additional files or data as belonging to that test case. Whenever the given {@link
-	 * TestData} changes, this test will be selected to be run by the TIA.
-	 * <p>
-	 * Example: For a test that reads test data from an XML file, you should pass the contents of that XML file as its
-	 * test data. Then, whenever the XML is modified, the corresponding test will be run by the TIA.
```

Comment:
Why did we remove the documentation comment? As that is "public API" of our teamscale-client which is used at customers we should keep the documentation.

Reply:
I did not remove the documentation comment but a whole unused function. See class before the changes:
`public class ClusteredTestDetails extends TestDetails {

	/**
	 * A unique identifier for the cluster this test should be prioritized within. If null the test gets assigned it's
	 * own unique cluster.
	 */
	public String clusterId;

	public ClusteredTestDetails(String uniformPath, String sourcePath, String content, String clusterId) {
	public String partition;

	public ClusteredTestDetails(String uniformPath, String sourcePath, String content, String clusterId,
								String partition) {
		super(uniformPath, sourcePath, content);
		this.clusterId = clusterId;
	}

	/**
	 * Creates clustered test details with the given additional {@link TestData}.
	 * <p>
	 * Use this to easily mark additional files or data as belonging to that test case. Whenever the given {@link
	 * TestData} changes, this test will be selected to be run by the TIA.
	 * <p>
	 * Example: For a test that reads test data from an XML file, you should pass the contents of that XML file as its
	 * test data. Then, whenever the XML is modified, the corresponding test will be run by the TIA.
	 */
	public static ClusteredTestDetails createWithTestData(String uniformPath, String sourcePath, TestData testData,
														  String clusterId) {
		return new ClusteredTestDetails(uniformPath, sourcePath, testData.hash, clusterId);
		this.partition = partition;
	}

}`

Reply:
Actually I would tend to keep the method + doc comment and use it in our custom test framework test as it was added by Fabi AFAIR for that use case